testnet/stacks-node/src/tests/mod.rs

```diff
@@ -439,31 +440,41 @@ fn should_succeed_mining_valid_txs() {
         match round {
             1 => {
                 // On round 1, publish the KV contract
-                tenure.mem_pool.submit_raw(&mut chainstate_copy, &consensus_hash, &header_hash, PUBLISH_CONTRACT.to_owned()).unwrap();
+                tenure.mem_pool.submit_raw(&mut chainstate_copy, &consensus_hash, &header_hash, PUBLISH_CONTRACT.to_owned(),
+                                &ExecutionCost::max_value()
+                ).unwrap();
             },
             2 => {
                 // On round 2, publish a "get:foo" transaction
                 // ./blockstack-cli --testnet contract-call 043ff5004e3d695060fa48ac94c96049b8c14ef441c50a184a6a3875d2a000f3 10 1 STGT7GSMZG7EA0TS6MVSKT5JC1DCDFGZWJJZXN8A store get-value -e \"foo\"
                 let get_foo = "8080000000040021a3c334fc0ee50359353799e8b2605ac6be1fe40000000000000001000000000000000a0100b7ff8b6c20c427b4f4f09c1ad7e50027e2b076b2ddc0ab55e64ef5ea3771dd4763a79bc5a2b1a79b72ce03dd146ccf24b84942d675a815819a8b85aa8065dfaa030200000000021a21a3c334fc0ee50359353799e8b2605ac6be1fe40573746f7265096765742d76616c7565000000010d00000003666f6f";
-                tenure.mem_pool.submit_raw(&mut chainstate_copy, &consensus_hash, &header_hash,hex_bytes(get_foo).unwrap().to_vec()).unwrap();
+                tenure.mem_pool.submit_raw(&mut chainstate_copy, &consensus_hash, &header_hash,hex_bytes(get_foo).unwrap().to_vec(),
+                                &ExecutionCost::max_value()
+                                           ).unwrap();
```

Comment:
Holy whitespace Batman! Does this past `cargo fmt`?

Reply:
I think `cargo fmt` specifically wanted this 🤷‍♂️

Reply:
Yeah seems like `cargo fmt` bug.

I would rather leave cases like this than start manually formatting code. But it can be manually formatted.

Reply:
Yeah, let's definitely leave cases like this